code/game/gamemodes/revolution/revsquad.dm

```diff
@@ -0,0 +1,309 @@
+//A variant of revolution, with an emphasis on a small group with co-ordinated efforts instead of greytiding
+
+#define REVSQUAD_FLASH_USES 1 // Number of times a specially spawned flash can convert normal crew members.
+/datum/game_mode/revsquad
+	name = "Revolution Squad"
+	config_tag = "revsquad"
+	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Mobile MMI","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer", "Internal Affairs Agent")
+
+	required_players = 4
+	required_players_secret = 25
+	required_enemies = 3
+	recommended_enemies = 3
+	var/finished = 0
+	var/checkwin_counter = 0
+	var/max_squaddies = 3
```

Comment:
Since this only used for the starting rev number, you should probably collapse it into recommended_enemies or required_enemies, that way people aren't editing two values.